### PR TITLE
Update Requirements

### PR DESCRIPTION
**Pull request by maximehuran**

### CODE REVIEW

**Review by jacquesbh — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
     "require": {
         "php": "~7.4",
         "sylius/sylius": "^1.8",
-        "gedmo/doctrine-extensions": "^2.4.12",
+        "gedmo/doctrine-extensions": "^2.4.12 || ^3.0",
         "monsieurbiz/sylius-rich-editor-plugin": "^2.0@RC",
-        "phpstan/phpstan-webmozart-assert": "^0.12.7"
+        "friendsofsymfony/oauth-server-bundle": "^1.6 || >2.0.0-alpha.0 ^2.0@dev"
```
Comment (jacquesbh) [inline composer.json:12]: Why?

File: tests/Application/templates [hunk 2]
```diff
@@ -0,0 +1 @@
+../../vendor/sylius/sylius/templates
```
Comment (jacquesbh) [inline tests/Application/templates:1]: Sure about this one?

**Review by jacquesbh — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (maximehuran) [reply composer.json:12]: Because it's needed to be compatible with Sylius 1.9 : https://github.com/Sylius/Sylius/blob/master/UPGRADE-1.9.md

Comment (maximehuran) [reply composer.json]: Moved to require-dev

**Review by jacquesbh — APPROVED** [review]

**Pull request merged**